content/PIFSC-opaka-length_M2.qmd

```diff
@@ -0,0 +1,461 @@
+---
+title: "Opakapaka Case Study"
+format: html
+---
+
+```{r fims1, warning=FALSE, message=FALSE}
+# automatically loads fims Rcpp module
+#remotes::install_github("NOAA-FIMS/FIMS", ref = "main")
+library(FIMS)
+library(TMB)
+library(dplyr)
+library(tidyr)
```

Comment:
All of these except r4ss are called in using setup.qmd so they are not needed here.